src/SDKs/DataMigration/AzSdk.RP.props

```diff
@@ -1,7 +1,7 @@
 ﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <!--This file and it's contents are updated at build time moving or editing might result in build failure. Take due deligence while editing this file-->
   <PropertyGroup>
-    <AzureApiTag>DataMigration_2018-03-15-preview;</AzureApiTag>
+    <AzureApiTag>DataMigration_2018-03-31-preview;DataMigration_2018-03-15-preview;</AzureApiTag>
```

Comment:
Apologize for the confusion, the earlier commit was correct, just needs one tag.